strapi/strapi
# feat: add document-actions to list-view

### What does it do?

* updates list-view to hold document actions
* tweaks document actions API to handle array of positions
* fixes issue with status badge in list-view that looked at wrong attribute
* fixes gap & alignment with long titles in edit-view
* fixes moving to published tab on publish

### How to test it?

* As a user I should be able to unpublish / discard draft & delete a document from the ListView
* As a user I should be able to navigate to the edit-view via the table actions button

### Related issue(s)/PR(s)

* resolves CONTENT-1927
